README.md

```diff
@@ -4,7 +4,7 @@ Swift Package Manager (SPM) package responsible for creating `Swift` code based
 
 Example usage and the expected spec of the `JSON` files can be found at `/Sources/MEGADesignToken`.
 
-Available enums (namespaces) containing tokens: `MEGADesignTokenDarkColors, MEGADesignTokenLightColors, MEGADesignTokenSpacing and MEGADesignTokenRadius`.
+Available enums (namespaces) containing tokens: `TokenColors, TokenSpacing and TokenRadius`.
```

Comment:
Also worth to mention that `dark/light` variation is handled automatically now

Reply:
Got it, will add it in the README